ts/pages/mesh/liquidity_market.tsx

```diff
@@ -0,0 +1,419 @@
+        orderSizeThreshold: 50,
+        getPrice: (order: SignedOrder, op: Operation) => {
+            let price;
+            const takerAssetAmount = new BigNumber(order.takerAssetAmount);
+            const makerAssetAmount = new BigNumber(order.makerAssetAmount);
+
+            // tslint:disable-next-line:switch-default
+            switch (op) {
+                case 'ask':
+                    price = takerAssetAmount.div(makerAssetAmount);
+                    break;
+                case 'bid':
+                    price = makerAssetAmount.div(takerAssetAmount);
+                    break;
+            }
+
+            return price;
+        },
+        getSize: (order: SignedOrder, op: Operation) => {
+            let size;
+            const takerAssetAmount = new BigNumber(order.takerAssetAmount);
+            const makerAssetAmount = new BigNumber(order.makerAssetAmount);
+            const constant = new BigNumber('1.0e18');
+
+            // tslint:disable-next-line:switch-default
+            switch (op) {
+                case 'ask':
+                    size = takerAssetAmount.div(constant);
+                    break;
+                case 'bid':
+                    size = makerAssetAmount.div(constant);
+            }
+
+            return size;
+        },
+    },
+    {
+        title: 'ETH/USDC',
+        quoteAsset: '0xf47261b0000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+        baseAsset: '0xf47261b0000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+        orderSizeThreshold: 50,
+        getPrice: (order: SignedOrder, op: Operation) => {
+            let price;
+            const takerAssetAmount = new BigNumber(order.takerAssetAmount);
+            const makerAssetAmount = new BigNumber(order.makerAssetAmount);
+            const constant = new BigNumber('1.0e12');
+
+            // tslint:disable-next-line:switch-default
+            switch (op) {
+                case 'ask':
+                    price = takerAssetAmount.div(makerAssetAmount).times(constant);
+                    break;
+                case 'bid':
+                    price = makerAssetAmount.div(takerAssetAmount).times(constant);
+                    break;
+            }
+
+            return price;
+        },
+        getSize: (order: SignedOrder, op: Operation) => {
+            let size;
+            const takerAssetAmount = new BigNumber(order.takerAssetAmount);
+            const makerAssetAmount = new BigNumber(order.makerAssetAmount);
+            const constant = new BigNumber('1.0e6');
+
+            // tslint:disable-next-line:switch-default
+            switch (op) {
+                case 'ask':
+                    size = takerAssetAmount.div(constant);
+                    break;
+                case 'bid':
+                    size = makerAssetAmount.div(constant);
+                    break;
+            }
+
+            return size;
+        },
```

Comment:
would modify the function to use remainingFillableTakerAssetAmount instead of makerAssetAmount/takerAssetAmounts.
The function needs to accept metadata (to be passed in the .map call in line 209)

I think it's best if I paste what I used locally, it seems to work better as insignificant orders get pruned.

```
getSize: (order: SignedOrder, metaData: any, op: Operation) => {
            let size;
            const takerAssetAmount = new BigNumber(order.takerAssetAmount);
            const makerAssetAmount = new BigNumber(order.makerAssetAmount);
            const remainingFillableTakerAssetAmount = new BigNumber(metaData.remainingFillableTakerAssetAmount);
            const constant = new BigNumber('1.0e18');

            // tslint:disable-next-line:switch-default
            switch (op) {
                case 'ask':
                    //size = takerAssetAmount.div(constant);
                    size = remainingFillableTakerAssetAmount.div(constant);
                    break;
                case 'bid':
                    //size = makerAssetAmount.div(constant);
                    size = remainingFillableTakerAssetAmount.div(takerAssetAmount).multipliedBy(makerAssetAmount).div(constant);
            }

            return size;
        },
```